src/Core/ObjectRegistry.php

```diff
@@ -66,25 +66,27 @@
      * @param array $config Additional settings to use when loading the object.
      * @return mixed
      */
-    public function load($objectName, $config = [])
+    public function load($objectName, array $config = [])
```

Comment:
Hmm.. maybe i was wrong or the cook book needs a correction.
http://book.cakephp.org/3.0/en/contributing/cakephp-coding-conventions.html#typehinting